db/migrate/20220709110049_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.integer :author_id
```

Comment:
- [ ] author_id should be a foreign key from the user table, Kindly reference it

Reply:
user_id actually. i've updated it appropriately. Thank you for the input.